### ISSUE TITLE
[VIP] Issue List with VIP transcoder plugin

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/transcoder#164]
@dhaval-parekh Please check the below list of issues.

- [x] **1. Shows the PHP Warning on `https://thumbnail-demo.dev3.rt.gw/wp-admin/admin.php?page=rt-transcoder` page when I enable the below settings on that page for the first time.**

- Over-write video thumbnails after retranscoding
- Allow admin to track real-time transcoding status on user profile

![Transcoder_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81145139-4e9b8180-8f93-11ea-83c6-ba900a3f6398.png)

-------

- [x] **2. .flv file type is not allowed.**

![Upload_New_Media_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81145857-cd44ee80-8f94-11ea-8136-da8d65b4ad16.png)

-----

- [x] **3. Getting the timeout notice with HTTP API Calls on transcoder settings page.**

https://thumbnail-demo.dev3.rt.gw/wp-admin/admin.php?page=rt-transcoder

![Transcoder_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81163094-d5f4ef00-8fab-11ea-828b-5277c3d721dd.png)

------

- [x] **4. Retransode functionality is completely broken.**

- Single media retranscoding ❌ 
- multiple media retranscoding ❌ 
- Bulk retranscoding ❌ 
- `Dashboard > Transcoder > Retranscde Media`  ❌ 

It always takes me to this page where I see this static information is placed. Also, the progress bar is missing.

![Retranscode_Media_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81165406-8c0e0800-8faf-11ea-896d-8a5657c266b5.png)

-------

- [x] **5. `Number of video thumbnails generated` - It always generates double number of thumbnails then given value.**

E.g Here I have set the value to `3` then after transcoding it shows 6 thumbnails

![Edit_Media_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81165595-dc856580-8faf-11ea-81aa-1e3ed62aa50f.png)

_NOTE: It seems it is generating duplicate thumbnails. Please check the highlighted part of the screenshot above._

**Comment by dharmin** [comment]
Fixed following on PR #165 

>   4. Retransode functionality is completely broken.
>  5. Number of video thumbnails generated - It always generates double number of thumbnails then given value.

**Comment by dharmin** [comment]
> 1 - Shows the PHP Warning on https://thumbnail-demo.dev3.rt.gw/wp-admin/admin.php?page=rt-transcoder page when I enable the below settings on that page for the first time.
> 3 - Getting the timeout notice with HTTP API Calls on transcoder settings page.

It's based on the time of server's response. If the server response within 3 seconds the error will not be shown up.

> 2 - .flv file type is not allowed.

.flv is not supported by VIP 
Ref.:
https://github.com/Automattic/vip-go-mu-plugins-built/blob/master/misc.php#L10-L13

**Comment by kjnanda** [comment]
@dhaval-parekh Please check the screenshot below. 

1. `Check Status` button should be visible once transcoding is done. It looks like a `regression` issue

![Media_Library_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81401197-71bd6100-914c-11ea-8e00-1741f0c351cb.png)

2.  Once the file is transcoded, it should show the message: `File is transcoded.` Not `Processed`

![Media_Library_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81401217-7c77f600-914c-11ea-96cd-b11d028b11b7.png)

3. `Thumbnail` generation seems not working completely.

![Media_Library_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81401446-f8723e00-914c-11ea-9220-b86c6dbbfb9f.png)

4. On the video media edit page, I don't see the video player at all! 🤔 

![Edit_Media_‹_thumbnail-demo_dev3_rt_gw_—_WordPress](https://user-images.githubusercontent.com/15173772/81401602-4be48c00-914d-11ea-878d-8550abc95b59.png)

4. Retranscode functionality still not working completely.

   1. Single media retranscoding  ✅ 

   2. multiple media retranscoding ✅ 

   3. Dashboard > Transcoder > Retranscde Media ❌ 
It still takes the static data

**Comment by dharmin** [comment]
> 1 - Check Status button should be visible once transcoding is done. It looks like a regression issue
> 2 - Thumbnail generation seems not working completely.
> 4 - On the video media edit page, I don't see the video player at all! 🤔
> 5 - Retranscode functionality still not working completely. Dashboard > Transcoder > Retranscde Media ❌ It still takes the static data

I'm not able to replicate these issues.

**Comment by kjnanda** [comment]
@dharmin I believe you resolved the issue for the `check status` button and we already replicate the `retranscode all` functionality where VIP only allows 10 media to retranscode.

**Comment by dhaval-parekh** [comment]
@kjnanda Could you please confirm is this issue is resolved (or we have separate issue)?
So we can close this issue.

**Comment by kjnanda** [comment]
@dhaval-parekh I am currently occupied with the task. Will check once done with priority task

**Comment by kjnanda** [comment]
@dhaval-parekh Issues mentioned in the task are done and working as expected.
